test/helpers/utils.go

```diff
@@ -491,14 +482,18 @@ func CreateMCP(t *testing.T, cs *framework.ClientSet, mcpName string) func() {
 	infraMCP.ObjectMeta.Labels = make(map[string]string)
 	infraMCP.ObjectMeta.Labels[mcpName] = ""
 	_, err := cs.MachineConfigPools().Create(context.TODO(), infraMCP, metav1.CreateOptions{})
-	if err != nil {
-		if !errors.IsAlreadyExists(err) {
-			require.Nil(t, err)
-		}
+	if err == nil {
+		t.Logf("Created MachineConfigPool %q", mcpName)
+	}
+	if errors.IsAlreadyExists(err) {
+		t.Logf("Reusing existing MachineConfigPool %q", mcpName)
+	} else {
+		require.NoError(t, err)
```

Comment:
well we know err exists here, sould this be `t.Fatalf`?

Reply:
Fixed!